includes/class-wc-google-gtag-js.php

```diff
@@ -263,7 +276,7 @@ public static function add_to_cart( WC_Product $product ) {
 		}
 
 		$event_code .= self::get_event_code(
-			'add_to_cart',
+			'add_to_cart_x',
```

Comment:
:question: 
Is that `_x` here on purpose? or just a leftover from debugging?
```suggestion
			'add_to_cart',
```

Reply:
Debugging 🤦 many thanks for catching! fe0ced4